Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -32,6 +33,22 @@ def check_for_unsupported_osx
         EOS
       end
 
+      def check_for_prerelease_xcodes
```

Comment:
There's only one Xcode we're checking, thus let's make this `check_for_prerelease_xcode` (without the trailing `s`).